frontend/app/view/codeeditor/schemaendpoints.ts

```
@@ -0,0 +1,38 @@
+
+type EndpointInfo = {
+    uri: string;
+    fileMatch: Array<string>;
+    schema: object;
+};
+
+const allFilepaths: Map<string, Array<string>> = new Map();
+allFilepaths.set(`${getWebServerEndpoint()}/schema/settings.json`, [`${getApi().getConfigDir()}/settings.json`]);
+
+async function getSchemaEndpointInfo(endpoint: string): Promise<EndpointInfo> {
+    let schema: Object;
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)